<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PngyuPreviewWindow</class>
 <widget class="QMainWindow" name="PngyuPreviewWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*left*/
QToolButton#toolButton_background_checkerboard,
QToolButton#toolButton_zoomout
{
  border-style: outset;
  border : 1px solid rgb(127,127,127,127);
  border-top-left-radius : 6px;
  border-bottom-left-radius: 6px;
  padding: 3px;
  background : qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0,
    stop:0 rgba(127, 127, 127, 64), stop:1 rgba(255, 255, 255, 100));
}

/*middle*/
QToolButton#toolButton_background_black
{
  border-style: outset;
  border : 1px solid rgb(127,127,127,127);
  border-radius : 0px;
  padding: 3px;
  background : qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0,
    stop:0 rgba(127, 127, 127, 64), stop:1 rgba(255, 255, 255, 100));
}

/*right*/
QToolButton#toolButton_background_white,
QToolButton#toolButton_zoomin
{
  border-style: outset;
  border : 1px solid rgb(127,127,127,127);
  border-top-right-radius : 6px;
  border-bottom-right-radius: 6px;
  padding: 3px;
  background : qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0,
    stop:0 rgba(127, 127, 127, 64), stop:1 rgba(255, 255, 255, 100));
}

QToolButton#toolButton_background_checkerboard::checked,
QToolButton#toolButton_background_black::checked,
QToolButton#toolButton_background_white::checked,
QToolButton#toolButton_zoomin::checked,
QToolButton#toolButton_zoomout::checked
{
  border : 1px solid rgb(100,100,100,127);
  border-style: inset;
  padding-top : 4px;
  padding-left : 4px;
  padding-bottom : 2px;
  padding-right : 2px;
  background : qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0,
    stop:1 rgba(40, 40, 40, 64), stop:0 rgba(127, 127, 127, 64));
}

QToolButton#toolButton_background_checkerboard::pressed,
QToolButton#toolButton_background_black::pressed,
QToolButton#toolButton_background_white::pressed,
QToolButton#toolButton_zoomin::pressed,
QToolButton#toolButton_zoomout::pressed
{ 
  border : 1px solid rgb(80,80,80,127);
  border-style: inset;
  padding-top : 4px;
  padding-left : 4px;
  padding-bottom : 2px;
  padding-right : 2px;
  background : qlineargradient(spread:reflect, x1:0, y1:1, x2:0, y2:0,
    stop:1 rgba(30, 30, 30, 64), stop:0 rgba(100, 100, 100, 64));
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>6</number>
    </property>
    <item row="0" column="6">
     <widget class="QWidget" name="widget_background_select" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QToolButton" name="toolButton_background_checkerboard">
         <property name="toolTip">
          <string>Set checker board pattern background</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/background/checkerboard.png</normaloff>:/background/checkerboard.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_background_select</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="toolButton_background_black">
         <property name="toolTip">
          <string>Set black pattern background</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/background/black.png</normaloff>:/background/black.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_background_select</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="toolButton_background_white">
         <property name="toolTip">
          <string>Set white pattern background</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/background/white.png</normaloff>:/background/white.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup_background_select</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox_show_original">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Show Original</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="7">
     <widget class="BasicImageView" name="imageview"/>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="pushButton_original_size">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Original Size (100%)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QWidget" name="widget_zoom" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="toolButton_zoomout">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/icons/zoomout.png</normaloff>:/icons/zoomout.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_zoomin">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../resource/resource.qrc">
           <normaloff>:/icons/zoomin.png</normaloff>:/icons/zoomin.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="SpinnerWidget" name="spinner" native="true">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BasicImageView</class>
   <extends>QGraphicsView</extends>
   <header>basicimageview.h</header>
  </customwidget>
  <customwidget>
   <class>SpinnerWidget</class>
   <extends>QWidget</extends>
   <header>spinnerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>checkBox_show_original</tabstop>
  <tabstop>toolButton_zoomout</tabstop>
  <tabstop>toolButton_zoomin</tabstop>
  <tabstop>toolButton_background_checkerboard</tabstop>
  <tabstop>toolButton_background_black</tabstop>
  <tabstop>toolButton_background_white</tabstop>
  <tabstop>imageview</tabstop>
  <tabstop>pushButton_original_size</tabstop>
 </tabstops>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_background_select"/>
 </buttongroups>
</ui>
